db/migrate/20220113132414_migrate_has_miscellaneous_information_in_claims.rb

```diff
@@ -0,0 +1,15 @@
+class MigrateHasMiscellaneousInformationInClaims < ActiveRecord::Migration[6.1]
+  class Claims < ActiveRecord::Base
+    self.table_name = :claims
+  end
+
+  def up
+    Claim.all.each do |claim|
+      claim.update_columns has_miscellaneous_information: claim.has_miscellaneous_information.present?
```

Comment:
This does not look right - please check with the original calculated value.  When you look at it, you are setting 'has_miscellaneous_information' based on 'has_miscellaneous_information' - which will  be nil as it is a new column !